apel/db/records/cloud.py

```diff
@@ -50,7 +50,10 @@ def __init__(self):
         self._db_fields = self._msg_fields[:9] + ['VO', 'VOGroup', 'VORole'] + self._msg_fields[9:]
         self._all_fields = self._db_fields
         
-        self._ignored_fields = ["UpdateTime"]
+        self._ignored_fields = [
+            "UpdateTime", "MeasurementTime", "MeasurementMonth",
```

Comment:
Annoying that it doesn't quite fit on one line. I'd be tempted to put month and year on the same line at least, and use non-hanging indentation like `_int_fields` otherwise we're doubling the number of lines.